src/components/Navbar/index.jsx

```diff
@@ -23,6 +23,8 @@ const Navbar = () => {
           await logoutService();
           dispatch(logout());
           window.localStorage.removeItem("token");
```

Comment:
move this to a common method and use it wherever needed

Reply:
Done!

Reply:
Done!